internal/cloud_connector/message_handlers_test.go

```diff
@@ -139,6 +144,8 @@ func TestHandleDuplicateAndOldOnlineMessages(t *testing.T) {
 	var connectedClientRecorder = &mockConnectedClientRecorder{}
 	var sourcesRecorder controller.SourcesRecorder
 
+	logger := logger.Log.WithFields(logrus.Fields{})
```

Comment:
I think we can remove this line and pass `logger.Log` into the `handleOnlineMessage()` function.